modules/mongodb/templates/mongodb-backup-s3.sh.erb

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+
+BACKUP_NODE=<%= @backup_node %>
+BACKUP_DIR=<%= @backup_dir %>
+BACKUP_FILE=mongodump-<%= @fqdn %>.$(date +%F_%T).tar.gz.gpg
+KEY_FINGERPRINT=<%= @private_gpg_key_fingerprint %>
+S3_BUCKET=<%= @s3_bucket %>
+
+
+exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
+
+function time_taken {
+  TIME="$(date +%s%N)"
+  $@
+  TIME="$(($(date +%s%N)-TIME))"
+  SECS="$((TIME/1000000000))"
+  MILLISECS="$((TIME/100000000))"
+  printf " FINISHED IN: %02d:%02d:%02d.%02d\n" "$((SECS/3600%24))" "$((SECS/60%60))" "$((SECS%60))" "${MILLISECS}"
+
+}
+
+echo "................STARTING BACKUP..................."
```

Comment:
I removed the echoes and the output just feels less useful.  We're printing the time it took but we're not sure what we're timing.  Just a thought.